Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -112,652 +711,840 @@ def remove_none_dict(input_dict: Dict) -> Dict:
-    demisto.incidents(incidents)
+def get_entity_types_command(client: ZFClient, args: Dict):
+    response_content: Dict = client.get_entity_types()
+    entity_types: List = response_content.get("results", [])
+    human_readable = []
+    for entity_type in entity_types:
+        type_name: str = entity_type.get("name", "")
+        type_id: int = entity_type.get("id", "")
+        human_readable.append({"Name": type_name, "ID": type_id})
+    headers = ["Name", "ID"]
+    return CommandResults(
+        outputs=entity_types,
+        readable_output=tableToMarkdown(
+            "ZeroFox Entity Types",
+            human_readable,
+            headers=headers,
+            removeNull=True,
+        ),
+        outputs_prefix="ZeroFox.EntityTypes",
+    )
 
 
-def test_module():
-    """
-    Performs basic get request to get item samples
-    """
-    get_policy_types()
-    demisto.results('ok')
+def get_policy_types_command(client: ZFClient, args: Dict):
+    response_content: Dict = client.get_policy_types()
+    policy_types: List = response_content.get("policies", [])
+    human_readable = []
+    for policy_type in policy_types:
+        type_name: str = policy_type.get("name", "")
+        type_id: int = policy_type.get("id", "")
+        human_readable.append({"Name": type_name, "ID": type_id})
+    headers = ["Name", "ID"]
+
+    return CommandResults(
+        outputs=policy_types,
+        readable_output=tableToMarkdown(
+            "ZeroFox Policy Types",
+            human_readable,
+            headers=headers,
+            removeNull=True,
+        ),
+        outputs_prefix="ZeroFox.PolicyTypes",
+        raw_response=response_content,
+    )
+
+
+def modify_alert_notes_command(client: ZFClient, args: Dict):
+    alert_id: int = dict_value_to_integer(args, "alert_id")
+    alert_notes: str = args.get("notes", "")
+    client.modify_alert_notes(alert_id, alert_notes)
+    response_content: Dict = client.get_alert(alert_id)
+    alert = response_content.get("alert", {})
+    contents = get_alert_contents(alert)
+
+    return CommandResults(
+        readable_output="Successful note modification of alert "
+                        f"with ID: {alert_id}",
+        outputs=contents,
+        outputs_prefix="ZeroFox.Alert",
+    )
+
+
+def submit_threat_command(client: ZFClient, args: Dict):
+    source: str = args.get("source", "")
+    alert_type: str = args.get("alert_type", "")
+    violation: str = args.get("violation", "")
+    entity_id: str = args.get("entity_id", "")
+    response_content: Dict = client.submit_threat(
+        source,
+        alert_type,
+        violation,
+        entity_id,
+    )
+    alert_id = response_content.get("alert_id")
+    output = f"Successful submission of threat. ID: {alert_id}."
+    return CommandResults(
+        readable_output=output,
+        raw_response=response_content,
+        outputs={"ID": alert_id},
```

Comment:
What is the purpose of returning the dicationary '{"ID": alert_id}'? If the fields of the alert that corresponds to 'alert_id' get changed, after running 'client.submit_threat', then it is better to return the whole alert object, using get_alert_contents(response_content).